src/modules/outfits/outfitSizePicker/outfitSizePicker.js

```diff
@@ -29,13 +29,15 @@ const OutfitSizePicker = styled(({
             <div className='roa-image-wrapper'>
               {renderProductLink(product)}
             </div>
-            <SizePicker
-              className='size-picker'
-              productId={product.id}
-              variants={product.variants}
-              currentSize={currentSize}
-              inputDisabled
-              onSizeSelect={onSizeSelect(product.id)} />
+            <div className='roa-price-size-box'>
```

Comment:
This isn't something I think we need to address now, but something I've been doing with styled components is breaking them into multiple styled components so like this would be

```
const RoaPriceSizeBox = styled.div`
  ...
`
```
and then you can use it as a component `<RoaPriceSizeBox>`, instead of having next css classes within a styled components. I think this is more readable and maintainable to modularize styled components this way.

Example here https://github.com/rocketsofawesome/quark-ui/blob/ea48729d1cb496c8654b71871d86ed2a85090427/src/ProductDisplayPage/ProductInformation/ProductInformation.js#L15

But like I said I think this is fine for now, but maybe not a pattern we want to continue using because it makes us right nested complex css in our components. When we could just write simple css for individual components